frontend/src/scenes/scenes.ts

```diff
@@ -53,6 +53,11 @@ export const sceneConfigurations: Record<Scene, SceneConfig> = {
         activityScope: ActivityScope.DASHBOARD,
         defaultDocsPath: '/docs/product-analytics/dashboards',
     },
+    [Scene.ErrorTracking]: {
+        projectBased: true,
+        name: 'Error tracking',
+        activityScope: ActivityScope.DASHBOARD,
```

Comment:
occurs to me I don't know what activityScope is here and I'm too lazy to look right now 🙈

Reply:
Copy / paste error. It has to do with the activity log - removing it until we implement the activity log for this product